jep/302/README.adoc

```diff
@@ -0,0 +1,260 @@
+= JEP-302: Evergreen snapshotting data safety system
```

Comment:
Fixed thanks. Didn't rename the folder though to avoid GitHub folding **all** comments below despite me not having addressed them yet. Will do when submitting upstream.

Reply:
I think GitHub is smart enough to handle rename detection, but I can understand not wanting to test that now!